modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase

```diff
@@ -2648,6 +2648,10 @@ definition {
 			key_pending = 1,
 			key_untracked = 0);
 
+		// Workaround for LPS-186021
```

Comment:
I didn't add the goToWorkflowSLASettings macro here because we don't need all of its steps in this test step